handbook/people-ops/spending-company-money.md

```diff
@@ -43,18 +43,8 @@ We want to make sure you have what you need to be productive and happy in your r
 
 **Software/SaaS/cloud:** please check with the appropriate team and manager to ask if we already have a subscription or licenses to the software. If not, please get @sourcegraph/people-ops's approval for software or a subscription that would cost more than $500 per year.
 
-**Internet/WiFi plan:** $50/month (no need to expense, this is automatically added onto your pay).
```

Comment:
I don't think these should be removed from this page. This page is probably where people go to see "what are all the reimbursable things".